src/tlo/methods/healthsystem.py

```diff
@@ -182,6 +182,7 @@ def __init__(self, module, *args, **kwargs):
         self._received_info_about_bed_days = None
         self.expected_time_requests = {}
         self.facility_info = None
+        self.used_equipment = set()
```

Comment:
to be consistent with other declarations that the HSI makes about what it requires, I think we should use capitals. I also think we don't need "used" as it's implicit.
```sugggetion:
self.EQUIPMENT = set()
```